### PR TITLE
Multiple Galeries using "data-gallery" attribute

### PR DESCRIPTION
**Pull request by funwithtriangles**
Implementation of multiple galleries using a "data-gallery" attribute for each anchor tag. Simply filters the "items" object upon click of anchor tag. Also, placeholders are cleared upon gallery close to prevent weird gallery crossover issues.

### CODE REVIEW

**Comment by davidklaw** [comment]
:thumbsup:

**Comment by funwithtriangles** [comment]
Galleries can now be defined by each item or by some ancestor, as suggested by @davidklaw in issue https://github.com/martinaglv/touchTouch/pull/5#issuecomment-10225246.

**Comment by davidklaw** [comment]
:thumbsup:

File: assets/touchTouch/touchTouch.jquery.js [hunk 1]
```diff
@@ -20,10 +21,11 @@
 	/* Creating the plugin */
 	
 	$.fn.touchTouch = function(){
-		
+
 		var placeholders = $([]),
 			index = 0,
-			items = this;
+			allitems = this;
```
Comment (martinaglv) [inline assets/touchTouch/touchTouch.jquery.js]: I don't think there should be a semi column here.

**Comment by martinaglv** [comment]
Looking good! Only a few minor suggestions - there is a semi column present that could introduce a bug, and an unnecessary console.log.

I also think that the example with multiple galleries shouldn't replace the old index.html. Maybe the two should coexist as example.html and example-multi.html?

**Comment by funwithtriangles** [comment]
Sorted out the semi-colon. You're probably right about the examples, there's no real tidy way to show it all in one page.

**Comment by martinaglv** [comment]
I will merge this request and will push the updated examples later today. Thank you for the patch!

**Comment by geekhunger** [comment]
still NOT doing it for me, when having the data-gallery attribute on the parent element:
< div class="thumbs" data-gallery="one" >...</div>
<div class="thumbs" data-gallery="two">...</div></pre>

**Pull request merged**